core/src/main/java/org/apache/gravitino/catalog/CatalogManager.java

```diff
@@ -260,7 +260,7 @@ private ModelCatalog asModels() {
 
   private final Config config;
 
-  @VisibleForTesting final Cache<NameIdentifier, CatalogWrapper> catalogCache;
+  @VisibleForTesting static Cache<NameIdentifier, CatalogWrapper> catalogCache;
```

Comment:
Why do we make it static? I assume there will be only one `CatalogManager`, so there should be only one `catalogCache`, right?

Reply:
> Why do we make it static? 

The method check `catalogInUse` and `metalakeInUse` are all static. If we want to use cache for them, we need to change it to `static`

 > assume there will be only one CatalogManager, so there should be only one catalogCache, right?

Yes, there will be only one cache and all catalogs shares the same instance, It's not a big problem I think.

Reply:
should named `CATALOG_CACHE`?

Reply:
should it be `final`?

Reply:
Okay, let me change it.

Reply:
If we use `static final` flag, then Gravitino server configuraoin like `gravitino.catalog.cache.evictionIntervalMs` should be remove as we can't used it in static code block, Is that acceptable to you?